src/presign/input.rs

```diff
@@ -1,3 +1,9 @@
+// Copyright (c) 2022-2023 Bolt Labs Holdings, Inc
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
```

Comment:
We might want to add a bash script to the CI that just checks for a copyright line for Bolt on every file so we don't miss stuff like this.